myp5.js

```diff
@@ -19,24 +19,20 @@ function setup() {
   textAlign(CENTER);
   textSize(50);
 
-  for (var i = 0; i < 100; i++) {
-    lyrics[i] = new lyrics(random(width), random(height));
-  }
 }
 
```

Comment:
You can't create your lyric objects in setup, you don't have data at this point. Instead, I created a function to create the data using the lyric data and call it when you have a successful ajax call